mantidimaging/core/operations/clip_values/clip_values.py

```diff
@@ -9,10 +9,10 @@
 
 
 class ClipValuesFilter(BaseFilter):
-    """Clips pixel values of the image based on the parameters. Can be used as
-    a way to mask a.
+    """Clips grey values of the image based on the parameters. Can be used to remove outliers
```

Comment:
I think **pixel values** is clearer than **grey values** but I will leave that to you. Just ensure consistency, it should be _grey_ or _gray_

Reply:
Winfried suggested "grey value", so I'll go with that.